boards/nxp/ucans32k1sic/ucans32k1sic-pinctrl.dtsi

```diff
@@ -74,4 +74,44 @@
 			drive-strength = "low";
 		};
 	};
+
+	adc0_default: adc0_default {
+		group0 {
+			pinmux = <ADC0_SE0_PTA0>,
+				<ADC0_SE1_PTA1>,
+				<ADC0_SE2_PTA6>,
+				<ADC0_SE3_PTA7>,
+				<ADC0_SE4_PTB0>,
+				<ADC0_SE5_PTB1>,
+				<ADC0_SE6_PTB2>,
+				<ADC0_SE7_PTB3>,
+				<ADC0_SE8_PTB13>,
+				<ADC0_SE8_PTC0>,
+				<ADC0_SE9_PTC1>,
+				<ADC0_SE10_PTC2>,
+				<ADC0_SE11_PTC3>,
+				<ADC0_SE12_PTC14>,
+				<ADC0_SE13_PTC15>,
+				<ADC0_SE14_PTC16>,
+				<ADC0_SE15_PTC17>;
+			drive-strength = "low";
+		};
+	};
+
+	adc1_default: adc1_default {
+		group0 {
+			pinmux = <ADC1_SE0_PTA2>,
+				<ADC1_SE1_PTA3>,
+				<ADC1_SE14_PTB0>,
+				<ADC1_SE15_PTB1>,
+				<ADC1_SE7_PTB12>,
+				<ADC1_SE8_PTB13>,
+				<ADC1_SE2_PTD2>,
+				<ADC1_SE3_PTD3>,
+				<ADC1_SE6_PTD4>,
+				<ADC1_SE10_PTE2>,
+				<ADC1_SE11_PTE6>;
```

Comment:
I don't see any ADC dedicated pin in the UCANS32K1SIC board schematics, hence there should not be default ADC pinmuxing for this board. But I'll let @PetervdPerk-NXP or @bperseghetti comment on this

Reply:
![image](https://github.com/user-attachments/assets/34557b33-4e59-4c45-99c3-b742993314b2)

![image](https://github.com/user-attachments/assets/667f1569-094d-46dd-99ae-12d62f700e6c)

PTE0 - Need to determine if there is an ADC channel that can mux here (i don't think so?)

Reply:
> I don't see any ADC dedicated pin in the UCANS32K1SIC board schematics, so AFAICT there should not be default ADC pinmuxing for this board. But I'll let @PetervdPerk-NXP or @bperseghetti comment on this

Hi, I’m looking for guidance on how to proceed. I understand that there aren’t dedicated ADC pins, but it’s possible to enable analog input on certain pins using alternate functions. Could it be that there’s just a minimal set of pins, such as PTC14 on ADC0, that can be used for this purpose?

Reply:
Board dts should assign pin functions according to the schematic, so if there's none assigned on this board, I don't see the purpose of having them defined. The user can always extend the pinmuxing by application dt overlays for those IOs that can be used with ADC functions